templates/pg_dump.sh.epp

```diff
@@ -0,0 +1,102 @@
+<%- if $facts['kernel'] == 'Linux' { -%>
```

Comment:
please add a comment that this file is managed by puppet

Reply:
Fixed.